tests/unit/mixins/test_fixtures.py

```diff
@@ -0,0 +1,36 @@
+from unittest.mock import create_autospec
+
+import pytest
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service.catalog import ColumnInfo, ColumnTypeName
+
+# Fixtures are used implicitly and ignored in test signatures using `# noqa: F811`
+# pylint: disable-next=unused-import
+from databricks.labs.ucx.mixins.fixtures import make_table, make_schema, make_random  # fmt: skip  # noqa: F401
+
+
+@pytest.fixture
+def ws():
+    workspace_client = create_autospec(WorkspaceClient)
+    yield workspace_client
+    # Circumvent linter warning; workspace client instance is never called at class level
+    workspace_client.assert_not_called()
```

Comment:
I'm having trouble figuring out where this is used…?

Reply:
Do you mean the fixture itself or the `assert_not_called`?

- Fixture itself: used implicitly by `make_table` fixture, see comment in imports
- The `assert_not_called` circumvents linter warning, see comment on line before

Reply:
Ah. I missed what's going on here:

 - The import of those methods effectively means the fixtures are declared within this module.
 - One of them (`make_table`) in turns needs a `ws` fixture to be available.
 - The parameter names on the test methods shadow the (global) imports, which is why you need those suppressions.